### PR TITLE
Header support

### PR DESCRIPTION
**Pull request by roxblnfk**
## What was changed

- Updated proto files to communicate with RR (added `header` field)
- Added `getHeader()` methods to Workflow and Activity facades
- Added `$header` parameter to Workflow/Activity stub factories
- Added passing for header values through Workflow/Activity executions
- Some fixes in psalm and phpunit configs

## Why?

Header supporting is required for interceptors implementation (#225)

## Additional

RoadRunner 2.12.2 is required.
Note: target branch is not the `master`

### CODE REVIEW

File: src/Workflow/WorkflowContextInterface.php [hunk 1]
```diff
@@ -174,20 +186,31 @@ public function executeChildWorkflow(
...
+     * @param HeaderInterface|array<non-empty-string, scalar>|null $header Optional header values.
+     *        The default {@see null} value means that header values will be inherited from the current context.
+     *
      * @return T
      */
-    public function newChildWorkflowStub(string $class, ChildWorkflowOptions $options = null): object;
+    public function newChildWorkflowStub(
+        string $class,
+        ChildWorkflowOptions $options = null,
+        HeaderInterface|array|null $header = null,
```
Comment (wolfy-j) [inline src/Workflow/WorkflowContextInterface.php:197]: do we actually need to expose it here?

Comment (wolfy-j) [reply src/Workflow/WorkflowContextInterface.php:197]: can you double check with Java SDK

**Review by wolfy-j — APPROVED** [review]

**Comment by wolfy-j** [comment]
This seems good. We have to make sure that we can properly update the SDK in relation to parent RR version.

File: src/Workflow/WorkflowContextInterface.php (hunk 1 above)
Comment (roxblnfk) [reply src/Workflow/WorkflowContextInterface.php:197]: > do we actually need to expose it here?

Why not?

**Review by wolfy-j — APPROVED** [review]

**Comment by bergundy** [comment]
We only expose headers to users via interceptors in our SDKs.
PHP shouldn't be special here.

**Pull request merged**